core/trino-spi/src/main/java/io/trino/spi/session/PropertyMetadata.java

```diff
@@ -120,7 +125,15 @@ public T getDefaultValue()
      */
     public boolean isHidden()
     {
-        return hidden;
+        return flags.contains(Flag.HIDDEN);
+    }
+
+    /**
+     * Is this property to be skipped in CREATE TABLE LIKE statements?
```

Comment:
Returns whether the property is not to be copied in in `CREATE TABLE LIKE INCLUDING PROPERTIES` statements